### PR TITLE
Add test for Carpool.get_ride_requests_query

### PR DESCRIPTION
**Pull request by drabinowitz**
Add tests for `Carpool` model

### CODE REVIEW

**Comment by iandees** [comment]
Yessss.

File: tests/test_models.py [hunk 1]
```diff
@@ -51,3 +51,53 @@ def test_roles(self):
...
+            ride_request_1,
+            ride_request_2,
+            ride_request_3,
+        ])
+        db.session.commit()
+
+        all_ride_requests = \
+            carpool_1.get_ride_requests_query().all()
+
+        assert len(all_ride_requests) == 2
```
Comment (jmcarp) [inline tests/test_models.py]: We can express this a little more concisely as

```
assert all_ride_requests == [ride_request_1, ride_request_2]
```

I'm also not sure we want to make assertions about order here. If we don't, we can use

```
assert set(all_ride_requests) == {ride_request_1, ride_request_2}
```

File: tests/test_models.py [hunk 2]
```diff
@@ -51,3 +51,53 @@ def test_roles(self):
...
+    """Carpool tests."""
+
+    def test_get_ride_requests_query(self, db):
+        """Test get ride requests query"""
+        carpool_1 = CarpoolFactory()
+        carpool_2 = CarpoolFactory()
+
+        person = PersonFactory()
+
+        ride_request_1 = RideRequest(
```
Comment (jmcarp) [inline tests/test_models.py]: To cut down on the boilerplate of instantiating a `Person`, we could implement this as a `RideRequestFactory` with a `SubFactory` to create the `Person`.

File: tests/test_models.py (hunk 1 above)
Comment (drabinowitz) [reply tests/test_models.py]: done

File: tests/test_models.py (hunk 2 above)
Comment (drabinowitz) [reply tests/test_models.py]: done

**Comment by drabinowitz** [comment]
@jmcarp this look good to you?

**Comment by jmcarp** [comment]
LGTM

**Pull request merged**